### ISSUE TITLE
采用swift infer 测试qwen2.5-omni模型结果，与官方测试方法结果不一致

### ISSUE THREAD

**Issue by yam741** [modelscope/ms-swift#4595]
请问swift对比qwen2.5-omni官方提供的cookbooks配置有哪里不一样嘛？改变system，infer也不会报官方的warning
WARNING:root:System prompt modified, audio output may not work as expected. Audio output mode only works when using default system prompt 'You are Qwen, a virtual human developed by the Qwen Team, Alibaba Group, capable of perceiving auditory and visual inputs, as well as generating text and speech.'

swift：
`swift infer --model $model --stream true --load_data_args true --max_new_tokens 2048 --val_dataset $val_jsonl --result_path $val_tmp_output`
其中val_jsonl内容：
 {"messages": [{"role": "system", "content": "You are a vocal sound classification model."}, {"role": "user", "content": "Listen to the given audio carefully and answer this question: Identify the human vocal sound in the audio.\nOptions:\n(A) Breath\n(B) Cough\n(C) Cry\n(D) Laugh\n(E) Scream\n(F) Sneeze\n(G) Yawn\n.Answer with the option's letter from the given choices directly and only give the best option."}, {"role": "assistant", "content": "Crying"}], "audios": ["59578-5_2_0.wav"]}

官方测试：
```
import random
import torch
from qwen_omni_utils import process_mm_info
from transformers import (Qwen2_5OmniForConditionalGeneration,Qwen2_5OmniProcessor)
model_path = $model同上
model = Qwen2_5OmniForConditionalGeneration.from_pretrained(
     model_path,
     device_map="auto",
 ).eval()
sys_prompt = "You are a vocal sound classification model."
task_prompt = "Listen to the given audio carefully and answer this question: Identify the human vocal sound in the audio.\nOptions:\n(A) Breath\n(B) Cough\n(C) Cry\n(D) Laugh\n(E) Scr    eam\n(F) Sneeze\n(G) Yawn\n.Answer with the option's letter from the given choices directly and only give the best option."

def inference(audio_path,sys_prompt,task_prompt):
     messages = [
         {"role": "system", "content": [{"type": "text", "text": sys_prompt}]},
         {"role": "user", "content": [
                 {"type": "text", "text": task_prompt},
                 {"type": "audio", "audio": audio_path},
             ]
        },
     ]
     text = processor.apply_chat_template(messages, tokenize=False, add_generation_prompt=True)
     audios, images, videos = process_mm_info(messages, use_audio_in_video=True)
     inputs = processor(text=text, audio=audios, images=images, videos=videos, return_tensors="pt", padding=True, use_audio_in_video=True)
     inputs = inputs.to(model.device).to(model.dtype) 
     output = model.generate(**inputs, use_audio_in_video=True, return_audio=False, thinker_do_sample=False)
     output = output[:, inputs.input_ids.size(1):]

     text = processor.batch_decode(output, skip_special_tokens=True, clean_up_tokenization_spaces=False)[0]
     return text
processor = Qwen2_5OmniProcessor.from_pretrained(model_path)
audio_path = "59578-5_2_0.wav"
response = inference(audio_path,sys_prompt,task_prompt)
print(response[0])
```

**Comment by Jintao-Huang** [comment]
swift中的model.generate 调用的是llm的generate

**Comment by Jintao-Huang** [comment]
结果不一致可以具体一些嘛

**Comment by yam741** [comment]
感谢您的回复，在做Nonspeech数据集测试时，官方测试方法结果ACC为68.83%，swift结果更好为82.34%
比如一条孩子在哭的音频，会因为激动部分时刻像在尖叫，官方测试方法结果是E Scream; swift结果为C Cry; label 为C；不清楚推理有哪些异同

**Comment by Jintao-Huang** [comment]
原始模型还是微调后的模型
